dbms/src/Common/iostream_debug_helpers.h

```diff
@@ -46,4 +46,4 @@ std::ostream & operator<<(std::ostream & stream, const DB::PODArray<T, INITIAL_S
 
 
 /// some operator<< should be declared before operator<<(... std::shared_ptr<>)
-#include <common/iostream_debug_helpers.h>
+#include <Common/iostream_debug_helpers.h>
```

Comment:
It was correct.
common is libs/libcommon/include/common 
Common is dbms/src/Common

Reply:
Ah I see, it's the same file name, fixed.

Reply:
Removed the GeoUtils.h patch from this PR as you ported it to https://github.com/yandex/ClickHouse/pull/1333 already